CALLING_TUTORIAL.md

```diff
@@ -86,4 +87,123 @@ hear the Twilio welcome message.
+      |> ExTwilio.Capability.allow_client_incoming("jenny")
+      |> ExTwilio.Capability.token
+
+    render(conn, "show.html", token: token)
+  end
+end
+```
+
+Create a view template for the `show` action
+
+```html
+<script type="text/javascript"
+  src="//media.twiliocdn.com/sdk/js/client/v1.3/twilio.min.js"></script>
+<script type="text/javascript"
+  src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js">
+</script>
+<script type="text/javascript">
+
+/* Create the Client with a Capability Token */
+Twilio.Device.setup("<%= @token %>", {debug: true});
+
+/* Let us know when the client is ready. */
+Twilio.Device.ready(function (device) {
+    $("#log").text("Ready");
+});
+
+/* Report any errors on the screen */
+Twilio.Device.error(function (error) {
+    $("#log").text("Error: " + error.message);
+});
+
+Twilio.Device.connect(function (conn) {
+    $("#log").text("Successfully established call");
+});
+
+/* Log a message when a call disconnects. */
+Twilio.Device.disconnect(function (conn) {
+    $("#log").text("Call ended");
+});
+
+/* Listen for incoming connections */
+Twilio.Device.incoming(function (conn) {
+  $("#log").text("Incoming connection from " + conn.parameters.From);
+  // accept the incoming connection and start two-way audio
+  conn.accept();
+});
+
+/* A function to end a connection to Twilio. */
+function hangup() {
+    Twilio.Device.disconnectAll();
+}
+</script>
+
+<!-- use an onclick action to hang up the call when the user presses
+the button -->
+<button class="hangup" onclick="hangup();">
+  Hangup
+</button>
+<div id="log"></div>
+```
+
+Next, we need to create a controller action that will render some TwiML instructing Twilio to connect any incoming calls to the client named _jenny_. There is a library called [ExTwiml](https://github.com/danielberkompas/ex_twiml) that provides a nice DSL for generating TwiML with Elixir. Add it as a dependency with `{:ex_twiml, "~> 2.1.0"}`.
+
+Create the following controller action and module for rendering the TwiML
+
+```elixir
+defmodule Demo.TwilioController do
+  use Demo.Web, :controller
+
+  def show(conn, _params) do
+    token =
+      ExTwilio.Capability.new
+      |> ExTwilio.Capability.allow_client_incoming("jenny")
+      |> ExTwilio.Capability.token
+
+    render(conn, "show.html", token: token)
+  end
+
+  # Note: By default, Twilio will POST to this endpoint
+  def voice(conn, _params) do
+    resp = Demo.Twiml.dial_jenny
+    conn
+    |> put_resp_content_type("text/xml")
+    |> text(resp)
+  end
+end
+
+defmodule Demo.Twiml do
+  import ExTwiml
+
+  def dial_jenny do
+    twiml do
+      # This should be your Twilio Number or verified Caller ID
+      dial callerid: "+1XXXXXXX" do
+        client "jenny"
+      end
+    end
+  end
+end
+```
+
+Once you've started your web server, you can expose your endpoint to the world with a tool like [Ngrok](https://ngrok.com/), pointing it to your development server on port 4000
+
+`ngrok http 4000`
```

Comment:
Would that make it a little bit clearer to people with no experience with ngrok, if this was formatted like CLI command?

Something like:

```
$ ngrok http 4000
```